gender_analysis/analysis/metadata_visualizations.py

```diff
@@ -1,27 +1,33 @@
+from pathlib import Path
+
 import matplotlib.pyplot as plt
 import seaborn as sns
+import os
 from collections import Counter
 
 from gender_analysis.common import MissingMetadataError
 
+DEFAULT_FILE_PATH = os.getcwd() + "/gender_analysis_visualizations/"
```

Comment:
I'd call this something like `VISUALIZATION_OUTPUT_DIR` so it's clearer what it is

Reply:
also, in general below instead of `file_path` let's refer to the `output_dir` or similar: "file path" usually means the entire path including the filename.